lib/govuk_app_config/govuk_error/sensitive_data_cleanser.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module SensitiveDataCleanser
+  def contains_sensitive_data?(string)
+    # Matches a rabbit mq url, which contains a password
+    /(amqps:\/\/[a-zA-Z0-9-_]+:)(.*)(@.*)/.match?(string)
+  end
+
+  def remove_sensitive_information(message)
+    # This strips rabbit mq passwords from their urls, to ensure we're not leaking credentials
+    # The regex splits text into three capture groups, something matching 'amqps://[host]:', the password,
+    # and then the rest of the url starting with '@'.
+    # The gsub then returns the first capture group unchange, the middle capture group (password) starred out,
+    # and the third capture group unchanged
+    message.gsub(/(amqps:\/\/[a-zA-Z0-9-_]+:)(.*)(@.*)/, '\1***\3')
+  end
+end
```

Comment:
The code looks fine, but I just wonder if the name of this module and/or these methods could be more specific. 
It doesn't strip _all_ sensitive information, just credentials, and only credentials given as `amqp_userinfo` in amqps URLs.

So maybe `contains_sensitive_data?` -> `contains_amqp_userinfo?` , and `remove_sensitive_information` -> `remove_amqp_userinfo` ?

Reply:
Also, the RegEx looks to be repeated twice. Maybe better to define it once as a constant at the top of this module?

Reply:
Yeah, good point, I guess I was originally thinking that it could be more generic in the future but I guess I'll leave that for when we need it